app/(candidate)/dashboard/voter-records/components/PaginationButtons.js

```diff
@@ -0,0 +1,31 @@
+export default function PaginationButtons({
+  currentPage,
+  totalPages,
+  onPageChange,
+}) {
+  if (totalPages <= 1) {
+    return null
+  }
```

Comment:
Sorry to nit-pick. But can be please remove this? It's redundant from the way it's rendered in the parent component, and there's a few different arguments as to why conditionally rendering a component or not should be done there, and not internally anyway.